uint8 role
uint8 id
float64 distance
float64  angle
float64 fp_rssi
float64 rx_rssi
bool time_out
